docs/tasks/run_jobs.md

```diff
@@ -0,0 +1,190 @@
+# Run Jobs
+
+This page show you how to run a Job in a Kubernetes cluster with Kueue enabled.
+
+The intended audience for this page are [batch users](/docs/tasks/README.md#as-a-batch-user)
+
+## Before you begin
+
+You need to have a Kubernetes cluster, the kubectl command-line tool
+must be configured to communicate with your cluster, and [Kueue installed](/README.md#installation).
+
+The cluster should have [quotas configured](administer_cluster_quotas.md).
+
+## 0. Identify the queues available in your namespace
+
+Run the following command to list the Queues available in your namespace.
+
+```shell
+kubectl -n default get queues
+```
+
+The output is similar to:
+
+```
+NAME   CLUSTERQUEUE    PENDING WORKLOADS
+main   cluster-total   3
+```
+
+The [ClusterQueue](/docs/concepts/cluster_queue.md) defines the quotas for the
+Queue.
+
+## 1. Define the Job
+
+Running a Job in Kueue is not much different from [running a Job in a Kubernetes cluster](https://kubernetes.io/docs/tasks/job/)
+without Kueue. The differences are:
+- You should create the Job in a [suspended state](https://kubernetes.io/docs/concepts/workloads/controllers/job/#suspending-a-job),
+  as Kueue will decide when it's the best time to start it.
+- You have to set the Queue you want to submit the Job to. This is done through
+  the annotation `kueue.x-k8s.io/queue-name`.
+- You should include the resource requests for each Job Pod.
+
+Here is a sample Job with three Pods that just sleep for a few seconds.
+This sample is also available in [config/samples/sample-job.yaml](/config/samples/sample-job.yaml).
+
+```yaml
+# sample-job.yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sample-job-
```

Comment:
Is it normal that the name ends with "-"?

Reply:
yes, because the system will add a string at the end, like: "sample-job-abcd"

Reply:
TIL this! Thanks.